### PR TITLE
Vng alias ip range go

### PR DESCRIPTION
**Pull request by pripatra**
_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket

_Include a link to your Jira Ticket_
_Example: [JIRAISS-1234](https://spotinst.atlassian.net/browse/JIRAISS-1234)_

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: service/ocean/providers/gcp/launchspec.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type LaunchSpec struct {
 	ResourceLimits         *ResourceLimits          `json:"resourceLimits,omitempty"`
 	LaunchSpecScheduling   *GKELaunchSpecScheduling `json:"scheduling,omitempty"`
 	LaunchSpecTags         []string                 `json:"tags,omitempty"`
+	VNGNetworkInterfaces   []*VNGNetworkInterface   `json:"networkInterfaces,omitempty"`
```
Comment (chandra1-n) [inline service/ocean/providers/gcp/launchspec.go:35]: name it to LaunchSpecNetworkInterfaces

File: service/ocean/providers/gcp/launchspec.go [hunk 2]
```diff
@@ -149,6 +150,32 @@ type GKELaunchSpecTaskHeadroom struct {
 	nullFields      []string
 }
 
+type VNGNetworkInterface struct {
+	VNGAccessConfigs []*VNGAccessConfig `json:"accessConfigs,omitempty"`
```
Comment (chandra1-n) [inline service/ocean/providers/gcp/launchspec.go:154]: change it to LaunchSpecAccessConfigs

File: service/ocean/providers/gcp/launchspec.go [hunk 3]
```diff
@@ -149,6 +150,32 @@ type GKELaunchSpecTaskHeadroom struct {
...
+	VNGAliasIPRanges []*VNGAliasIPRange `json:"aliasIpRanges,omitempty"`
```
Comment (chandra1-n) [inline service/ocean/providers/gcp/launchspec.go:155]: change it to LaunchSpecAliasIPRanges

File: service/ocean/providers/gcp/launchspec.go [hunk 4]
```diff
@@ -149,6 +150,32 @@ type GKELaunchSpecTaskHeadroom struct {
...
+type VNGAccessConfig struct {
+	VNGName *string `json:"name,omitempty"`
+	Type    *string `json:"type,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type VNGAliasIPRange struct {
+	IPCIDRRange         *string `json:"ipCidrRange,omitempty"`
```
Comment (chandra1-n) [inline service/ocean/providers/gcp/launchspec.go:172]: change to IPCidrRange

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix review comments

File: service/ocean/providers/gcp/launchspec.go (hunk 1 above)
Comment (sharadkesarwani) [reply service/ocean/providers/gcp/launchspec.go:35]: Updated to LaunchSpecNetworkInterfaces.

File: service/ocean/providers/gcp/launchspec.go (hunk 2 above)
Comment (sharadkesarwani) [reply service/ocean/providers/gcp/launchspec.go:154]: Updated to LaunchSpecAccessConfigs

File: service/ocean/providers/gcp/launchspec.go (hunk 3 above)
Comment (sharadkesarwani) [reply service/ocean/providers/gcp/launchspec.go:155]: Updated to LaunchSpecAliasIPRanges

File: service/ocean/providers/gcp/launchspec.go (hunk 4 above)
Comment (sharadkesarwani) [reply service/ocean/providers/gcp/launchspec.go:172]: update to IPCidrRange.

**Pull request merged**